server/middlewares/auth.js

```diff
@@ -1,25 +1,26 @@
+
 const jwt = require('jsonwebtoken');
 
 const secret = process.env.SECRET;
 
 
 const verifyToken = (req, res, next) => {
   const token = req.headers.authorization;
-    if (token) {
-        jwt.verify(token, secret, (err, decoded) => {
-            if (err) {
-            res.json({ success: false, message: 'Could not authenticate token.' });
-            } else {
-            req.decoded = decoded;
-            next();
-            }
-        });
-    } else {
-        return res.status(403).send({
-            status: 'Failed',
-            message: 'No token provided.'
-        });
-    }
+  if (token) {
+    jwt.verify(token, secret, (err, decoded) => {
+      if (err) {
+        res.json({ success: false, message: 'Could not authenticate token.' });
+      } else {
```

Comment:
I'll say that after decoding the token, you should use the user id to query the user's table and confirm that the user exists before sending him/her to the next route